Modix.Data/Repositories/MessageRepository.cs

```diff
@@ -141,5 +144,16 @@ private IQueryable<MessageEntity> GetGuildUserMessages(ulong guildId, ulong user
             return ModixContext.Messages.AsNoTracking()
                 .Where(x => x.GuildId == guildId && x.AuthorId == userId && x.Timestamp >= earliestDateTime);
         }
+
+        public async Task<MessageEntity> GetMessage(ulong messageId)
+        {
+            return await ModixContext.Messages.FindAsync(messageId);
+        }
+
+        public void UpdateStarboardColumn(MessageEntity message)
```

Comment:
Also, this method can be async and use the XAsync variants here.

Reply:
Since we're only updating 1 column I felt like it wouldn't really matter. But if you really want me to ;)

Reply:
I accidentally marked this as resolved before I actually pushed the changes, nevertheless, it's async now.